src/main/java/org/mongounit/MongoUnitUtil.java

```diff
@@ -261,13 +260,9 @@ private static List<String> getCollectionNamesToUse(
    */
   private static List<String> getCollectionNames(MongoDatabase mongoDatabase) {
 
-    List<String> collectionNames = new ArrayList<>();
-
     // Retrieve collection names from db
-    for (String collectionName : mongoDatabase.listCollectionNames()) {
-      collectionNames.add(collectionName);
-    }
-
+    List<String> collectionNames = new ArrayList<>();
+    mongoDatabase.listCollectionNames().forEach(collectionNames::add);
```

Comment:
@alexisGandar  Same question here. Just using newer syntax, but doesn't have to do with the intention of the PR, correct?